src/components/welcome/Welcome.jsx

```diff
@@ -0,0 +1,132 @@
+import { Button, Box, Typography } from "@mui/material";
+import React, { useState } from "react";
+import Tour from "reactour";
+import Buttons from "./Buttons";
+import QuestionMarkIcon from "@mui/icons-material/QuestionMark";
+
+const Welcome = () => {
+  const steps = [
+    {
+      style: {
+        minWidth: "750px",
+      },
+      content: () => (
+        <Box
+          sx={{
+            minWidth: "500px",
+          }}
+        >
+          <div style={{ display: "flex" }}>
+            <img
+              alt="logo"
+              style={{
+                maxWidth: "50px",
+                height: "auto",
+                flex: "20%",
+              }}
+              src={require("../../assets/logo.png")}
+            />
+            <Typography
+              variant="h6"
+              component="h2"
+              sx={{ flex: "80%", marginTop: "8px" }}
+            >
+              Welcome to Hunger Games
+            </Typography>
+          </div>
+          <Typography sx={{ mt: 2 }} component="p">
+            Hunger Games is a collection of mini-games that help contribute to
+            Open Food Facts in many ways. You can play are really exciting games
+            from anywhere. Here is a guided tour of the entire website which
+            will get you started.
+          </Typography>
+          <Typography sx={{ mt: 2 }} component="p">
+            Make sure to login to the Open Food Facts website to get credit for
+            your contributions. If you make a mistake, please take the time to
+            correct it by heading to the website.
+          </Typography>
+          <Typography sx={{ mt: 2 }} component="p">
+            In case of any doubt about the games, click on the question mark
+            icon at the top right of the page, and you can reach us on slack at
+            the #hunger-games channel
+          </Typography>
+          <hr />
```

Comment:
if need some additional space I think is better to define it using css, for example wrapping the `Buttons` in a `Box` and adding some padding or margin on top